<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutboundWidget</class>
 <widget class="QWidget" name="OutboundWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="balancersWidget" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>197</width>
     <height>150</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="balancerLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="balancerSelectionCombo">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QListWidget" name="balancerList"/>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="balancerDelBtn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="balancerAddBtn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
